server/config/v2_deprecation.go

```diff
@@ -24,10 +24,10 @@ const (
 	V2_DEPR_1_WRITE_ONLY = V2DeprecationEnum("write-only")
 	// V2store is WIPED if found !!!
 	V2_DEPR_1_WRITE_ONLY_DROP = V2DeprecationEnum("write-only-drop-data")
-	// V2store is neither written nor read. Usage of this configuration is blocking
-	// ability to rollback to etcd v3.5.
+	// V2store is neither written nor read for 3.6. v2snapshot is published only for backward compatibility
 	V2_DEPR_2_GONE = V2DeprecationEnum("gone")
 
+	//TODO geetasg does thie default need to change
```

Comment:
By implementing v2snapshots from v3 membership data, you basically implement the `write-only-drop-data`. 
I would be for totally implementing this control flags as it gives false information on what is really supported. 

Maybe we should improve documentation and explain:
* etcd v3.4.X - supports both v2, and v3 API.
* etcd v3.5.X - supports v3 api by default, v2 API needs to be separably enabled.
* etcd v3.6.X - supports only v3 api, v2 API is not enabled and any v2 data will be purged. 

Note; upgrading from v3.5 to v3.6 and then downgrading back to v3.5 will be supported, but v2 data will be lost.  cc @ptabor @ahrtr

Reply:
sg. I will change the default to write-only-drop-data and update the docs when ready.